index.yaml

```diff
@@ -8,13 +8,13 @@ distributions:
       overlay: crystal/ci-overlay.yaml
     doc_builds: {}
     notification_emails:
-      - steven+build.ros2.org@openrobotics.org
-      - ros2-buildfarm-crystal@googlegroups.com
+      - jrivero+build.ros2.org@openrobotics.org
     release_builds:
       default: crystal/release-build.yaml
       ubv8: crystal/release-bionic-arm64-build.yaml
     source_builds:
       default: crystal/source-build.yaml
+      gpu: crystal/source-gpu-build.yaml
```

Comment:
Since Crystal is effectively EOL please remove Crystal specific changes from this PR.

Reply:
03c39ee